src/main/java/executor/service/service/webdriver/WebDriverInitializerImpl.java

```diff
@@ -0,0 +1,65 @@
+package executor.service.service.webdriver;
+
+import com.google.common.base.Strings;
+import executor.service.config.properties.PropertiesConstants;
+import executor.service.model.configs.WebDriverConfig;
+import executor.service.model.proxy.ProxyConfigHolder;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.io.File;
+import java.time.Duration;
+
+
+public class WebDriverInitializerImpl implements WebDriverInitializer {
+
+    private final WebDriverConfig webDriverConfig;
+    public WebDriverInitializerImpl(WebDriverConfig webDriverConfig) {
+        this.webDriverConfig = webDriverConfig;
+    }
+
+    @Override
+    public WebDriver getInstance(ProxyConfigHolder proxyConfigHolder) {
```

Comment:
Maybe it's better to break this healthy method into smaller ones or make it more simplified?

Reply:
@git-alexbondarenko Can your answer for this question?